Bulk delete change (#2020)
Rapptz/discord.py: docs/api.rst

.. function:: on_message_delete(message)

    Called when a message is deleted. If the message is not found in the
    internal message cache, then this event will not be called. This
    happens if the message is too old or the client is participating in high
    traffic guilds. To fix this, increase the ``max_messages`` option of
    :class:`Client`.

    :param message: A :class:`Message` of the deleted message.

.. function:: on_bulk_message_delete(messages)

    Called when messages are bulk deleted. If none of the messages deleted
    are found in the internal message cache, then this event will not be called.

>     If some individual messages are not found in the internal message cache,

Reviewer: ```suggestion
    If individual messages were not found in the internal message cache,
```

>     the event will still be called, but the messages will not be included in

Reviewer: ```suggestion
    this event will still be called, but the messages not found will not be included in
```

>     the messages list. Messages will not be in cache if the message is too old

Reviewer: ```suggestion
    the messages list. Messages might not be in cache if the message is too old
```

Wording is too strong.

    or the client is participating in high traffic guilds.

>     To fix this, increase the ``max_messages`` option of :class:`Client`.

Reviewer: ```suggestion

    If this occurs increase the :attr:`Client.max_messages` attribute.
```

Needs a newline as well.

Author: Should I update existing documentation to follow this? Also, why a newline here?

Reviewer: Obviously.

Newline because it stands out.

>     :param messages: A List[:class:`Message`] of the deleted messages.

Reviewer: ```suggestion
    :param messages: A :class:`list` of :class:`Message` that have been deleted.
```
